pages/about/our-team/index.js

```diff
@@ -4,20 +4,40 @@ import Card from '@material-ui/core/Card';
 import CardMedia from '@material-ui/core/CardMedia';
 import Typography from '@material-ui/core/Typography';
 import CardContent from '@material-ui/core/CardContent';
-import Grid from '@material-ui/core/Grid';
 import GridList from '@material-ui/core/GridList';
 import GridListTile from '@material-ui/core/GridListTile';
-import Paper from '@material-ui/core/Paper';
 import Slide from '@material-ui/core/Slide';
 
 import AboutNav from '../../../components/AboutNav'
 import ImageCard from '../../../components/ImageCard'
 import Navigation from '../../../components/Navigation'
 import Footer from '../../../components/Footer'
 import {useStyles} from '../../../styles/AboutStyles'
+import useMediaQuery from '@material-ui/core/useMediaQuery';
 
 export default function Services() {
     const classes = useStyles();
+    const theme = useTheme();
+
+    const sm = useMediaQuery(theme.breakpoints.down('sm'));
+    const md = useMediaQuery(theme.breakpoints.down('md'));
+    const xlg = useMediaQuery(theme.breakpoints.down('xl'));
+
+    const mdUp = useMediaQuery(theme.breakpoints.up('md'));
+    const ipad = useMediaQuery(theme.breakpoints.down(769));
+    const ipadUp = useMediaQuery(theme.breakpoints.up(768));
+
+    const numCols = () => {
+        if (ipad && ipadUp) {
+          return 2;
+        } else if (sm) {
```

Comment:
I know I did the same because I am big dumb, but this `else if (sm)` and `else if (md)` can be merged into a `else if (sm || md)`

Reply:
will change now